tests/functional_tests/ffront_tests/test_execution.py

```diff
@@ -1164,7 +1164,7 @@ def program_domain(
         lower_j: int64,
         upper_j: int64,
     ):
-        fieldop_domain(a, out=a, domain={IDim: (lower_i, upper_i), JDim: (lower_j, upper_j)})
+        fieldop_domain(a, out=a, domain={IDim: (lower_i - 1, upper_i), JDim: (lower_j, upper_j)})
```

Comment:
This would require implementing the calls to them and will be problematic with the c++ backend, as they might return floating point types for integer arguments (which shouldn't be the case, but we never implemented this).